examples/vkcs-blockstorage-volume-create/main.tf

```diff
@@ -0,0 +1,10 @@
+resource "vkcs_blockstorage_volume" "volume" {
+  name = "volume"
+  description = "test volume"
+  metadata = {
```

Comment:
probably better to use tags? tags are shown on front

Reply:
Volumes don't have tags